pallets/funding/src/traits.rs

```diff
@@ -32,6 +32,22 @@ pub trait ProvideAssetPrice {
 	type AssetId;
 	type Price: FixedPointNumber;
 	fn get_price(asset_id: Self::AssetId) -> Option<Self::Price>;
+
+	fn calculate_decimals_aware_price(
```

Comment:
Could you add some comments with an calculation example explaining what this function does? Also for get_decimals_aware_price?